firefox-ios/firefox-ios-tests/Tests/XCUITests/PocketTests.swift

```diff
@@ -5,8 +5,49 @@
 import XCTest
 
 class PocketTests: BaseTestCase {
+    enum SwipeDirection {
+        case up, down, left, right
+    }
+
+    func validatePocketStoriesCount() {
+        let numPocketStories = app.collectionViews.containing(
+            .cell,
+            identifier: AccessibilityIdentifiers.FirefoxHomepage.Pocket.itemCell
+        ).children(matching: .cell).count - 1
+
+        if iPad() {
+            XCTAssertTrue(numPocketStories > 7, "Expected at least 8 stories on iPad.")
+        } else {
+            XCTAssertTrue(numPocketStories > 6, "Expected at least 7 stories on iPhone.")
+        }
+    }
+
+    func togglePocket(shouldEnable: Bool) {
+        navigator.performAction(shouldEnable ? Action.TogglePocketInNewTab : Action.TogglePocketInNewTab)
+        navigator.goto(NewTabScreen)
+    }
+
+    func scrollToElement(_ element: XCUIElement, direction: SwipeDirection, maxSwipes: Int) {
+        var swipeCount = 0
+        while !element.exists && swipeCount < maxSwipes {
+            switch direction {
+            case .up:
+                app.swipeUp()
+            case .down:
+                app.swipeDown()
+            case .left:
+                app.swipeLeft()
+            case .right:
+                app.swipeRight()
+            }
+            swipeCount += 1
+        }
+        XCTAssertTrue(element.exists, "Element \(element) not found after \(maxSwipes) swipes.")
+    }
+
     // https://mozilla.testrail.io/index.php?/cases/view/2306924
     func testPocketEnabledByDefault() {
+        let maxSwipes = 5
```

Comment:
We can set `5` as the default for `maxSwipes`. See how we set the default `TIMEOUT` of `mozWaitForElementToExist` from `BaseTestCase.swift`.